site/src/contexts/useProxyLatency.ts

```diff
@@ -15,6 +15,9 @@ export interface ProxyLatencyReport {
 	latencyMS: number;
 	// at is when the latency was recorded.
 	at: Date;
+	// nextHopProtocol can determine if HTTP/2 is being used.
+	// https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming/nextHopProtocol
```

Comment:
this might be a nice `/** doc comment */`

Reply:
I will never get used to doc comments coming from Go :laughing: